java/src/main/native/src/AggregationJni.cpp

```diff
@@ -82,9 +82,9 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_Aggregation_createNoParamAgg(JNIEnv
         case 27: // MERGE_M2
           return cudf::make_merge_m2_aggregation();
         case 28: // RANK
-          return cudf::make_rank_aggregation();
+          return cudf::make_rank_aggregation(cudf::rank_method::MIN, {}, null_policy::INCLUDE);
```

Comment:
The following should fix the Java build failure.
```suggestion
          return cudf::make_rank_aggregation(cudf::rank_method::MIN, {}, cudf::null_policy::INCLUDE);
```